docs/tasks/timeout-injection.md

```diff
@@ -0,0 +1,116 @@
+# Injecting Timeouts
```

Comment:
This document needs to be updated to fit in istio.io's model. You can read about what's needed here: https://istio.io/docs/home/contribute/writing-a-new-topic.html

Basically, the doc needs to include Jekyll front-matter, needs to use the templating mechanism, and the document needs to be registered in the navigation side-bar file.

Thanks.